<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>92</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>티머니 둥근바람 Regular</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="project_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detail_pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>속성</string>
   </property>
  </widget>
  <widget class="QComboBox" name="status_choose_comboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Release</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Testing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radar Only</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="apply_pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>적용</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backtest_pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back-Tesing</string>
   </property>
  </widget>
  <widget class="QLabel" name="algorithms_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="radar_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Algorithm : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radar : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
